README.md

```diff
@@ -1538,6 +1538,36 @@ For an example, simply visit the `out` directory and run following command to de
 now
 ```
 
+### Copying custom files
+
+In case you have to copy custom files like a robots.txt or generate a sitemap.xml you can do this inside of `exportPathMap`.
+`exportPathMap` gets a few contextual parameter to aid you with creating/copying files: 
+
+- `dev` - `true` when `exportPathMap` is being called in development. `false` when running `next export`. In development `exportPathMap` is used to define routes and behavior like copying files is not required.
```

Comment:
Didn't know that you can run `next export` in devmode 🤔. This needs to be implemented, right?

Reply:
You can't. `exportPathMap` runs when it's defined when using `next` / booting in dev mode.